darwin/datatypes.py

```diff
@@ -547,6 +550,9 @@ class AnnotationFile:
     #: List of ``VideoAnnotation``\s or ``Annotation``\s.
     annotations: Sequence[Union[Annotation, VideoAnnotation]]
 
+    # Item-level properties
+    item_properties: Optional[List[Dict[str, str]]] = None
```

Comment:
any reason we're using List, Dict rather than list, dict like for property_values?

Reply:
No - This makes more sense